.github/workflows/test.yml

```diff
@@ -134,6 +134,10 @@ jobs:
 
       - name: Login to Docker Hub
         uses: docker/login-action@v2
+        env:
```

Comment:
https://stackoverflow.com/questions/70249519/how-to-check-if-a-secret-variable-is-empty-in-if-conditional-github-actions

Reply:
It's weird but 🤷🏼 if that works ✔️